File: docs/snippets/all/archetypes/boxes3d_simple.py
"""Log a single 3D Box."""

import rerun as rr

rr.init("rerun_example_box3d", spawn=True)

rr.log("simple", rr.Boxes3D(half_sizes=[2.0, 2.0, 1.0]), rr.InstancePoses3D(translations=[1, 2, 3]))

File: docs/snippets/all/archetypes/boxes3d_simple.cpp
// Log a single 3D box.

#include <rerun.hpp>

int main(int argc, char* argv[]) {
    const auto rec = rerun::RecordingStream("rerun_example_box3d");
    rec.spawn().exit_on_failure();

    rec.log("simple", rerun::Boxes3D::from_half_sizes({{2.f, 2.f, 1.0f}}));
}
